src/include/migraphx/apply_alpha_beta.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+#define MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+
+#include "migraphx/errors.hpp"
+#include <migraphx/instruction.hpp>
+#include <migraphx/instruction_ref.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/common.hpp>
+#include <migraphx/module.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+template <typename T = float>
+instruction_ref insert_apply_alpha_beta(module& m,
+                                        instruction_ref pos,
+                                        const std::vector<instruction_ref>& args,
+                                        const std::string& op_name,
```

Comment:
If we want to use this for other operators like convolution or if we later add attributes to be set on `dot`, then we accept an `operation` rather than an op name.

Reply:
Can the parameter be changed to accept `operation` instead of just the op_name?

Reply:
Done